src/carousel.scss

```diff
@@ -21,6 +21,14 @@ $button: #{$fd-namespace}-button;
   @include fd-reset();
   @include fd-flex(column);
 
+  @include fd-hover() {
+    .#{$block}__content {
+      .#{$block}__button.#{$button} {
```

Comment:
Do you need `.#{$button}`? You can avoid this by adding more specificity.

Reply:
`.#{$button}` not needed. It works fine with only `.#{$block}__button`.
corrected.